src/components/KCard/KCard.vue

```diff
@@ -75,8 +75,15 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, computed } from 'vue'
+import { defineComponent, computed, PropType } from 'vue'
 import { v1 as uuidv1 } from 'uuid'
+import type { BorderVariant, BorderVariantRecord } from '@/types/card'
```

Comment:
Import from the root
```suggestion
import type { BorderVariant, BorderVariantRecord } from '@/types'
```